pkg/identity/cache/local.go

```diff
@@ -55,6 +55,7 @@ func (l *localIdentityCache) getNextFreeNumericIdentity(idCandidate identity.Num
 			log.Debugf("Reallocated restored CIDR identity: %d", idCandidate)
 			return idCandidate, nil
 		}
+		log.Warnf("Could not restore CIDR identity: %d", idCandidate)
```

Comment:
This could emit _thousands_ of logs right? I'm not sure if logging once per identity is appropriate.

Even if it is, this should use structured logging so that dynamic content doesn't increase the cardinality of the logs when they're aggregated in some logging system:
```suggestion
		log.WithField(logfields.Identity, idCandidate).Warn("Could not restore CIDR identity")
```

Reply:
It could, but it really should not! Only if we broke something so that the local allocator already has competing allocations made. I'll change this to structured logging.

Reply:
Also moved this logging to the daemon bootstrap code and made it to log only the first missed old CIDR identity so that we don't accidentally log thousands of logs.